docs/dataformats/observation_lists.rst

```diff
@@ -22,23 +22,48 @@ This has the advantage that everyone can work with whatever tool they like. Here
 * `csvkit <https://csvkit.readthedocs.org/en/latest/>`_
 * Your own script using e.g the `Python standard library CSV module <http://docs.python.org/2/library/csv.html>`_ or `pandas <http://pandas.pydata.org>`_
 
-A run list must have at least a column called ``Run``::
+A run list must have at least a column called ``OBS_ID``::
  
-   Run
+   OBS_ID
    42
    43
 
-Usually it has many more columns with information about each run::
- 
-   Run,Telescope_Pattern,Date,Duration,GLON,GLAT,Target,Zenith
-   1234,24,2013-03-22 14:32,1832,83.7,-5.2,Crab Nebula,32
+Usually it has many more columns with information about each observation. A list of Gammapy supported columns is::
+
+   OBS_ID: observation ID as an integer (starting at 0 or 1? does it matter?)
+   RA: pointing position right ascension in equatorial (ICRS) coordinates
+   DEC: pointing position declination in equatorial (ICRS) coordinates
+   AZ: average azimuth angle during the observarion
+   ALT: average altitude angle during the observarion
+   MUON_EFFICIENCY: average muon efficiency of the telescopes
+   TIME_START: start time of the observation stored as number of seconds after the time reference in the header
```

Comment:
I don't know if RST tables can nicely accomodate such long descriptions. If they automatically wrap to multiple rows or there's a way to do that, great. If no, please put a short description here and then a bullet list with longer explanations on some things below the table.